single-audit/resolve_findings/templates/resolve_findings/finding-resolution-page.html

```diff
@@ -112,14 +112,49 @@ <h2 class="usa-accordion__heading">
           <span class="text-uppercase text-bold text-white">
             Communication
           </span>
-          <span class="usa-tag margin-x-05">3 comments</span>
+
+          {% with comments.count as total_comments %}
+            <span class="usa-tag margin-x-05">
+              {{ total_comments }} comment{{ total_comments|pluralize }}
+            </span>
+          {% endwith %}
         </div>
+
+        <!-- @alexsoble, what would you think of this subsection (comments)
+             having its own div? -->
+        {% for comment in comments %}
+          <div class="comment_summary padding-x-205 padding-y-105 clearfix">
+            <!-- @todo (@fureigh): Format/transform publication date appropriately. -->
+            <!-- @todo (@fureigh): Extract agency name from 'author' (User).
+                 Requires first either 1) adding that field or 2) (preferable in
+                 the long run:) incorporating Login.gov and reverse engineering
+                 it from the email address. -->
+            <div>
+              <span class="text-bold">{{ comment.author }}</span> | DHS | {{ comment.created }}
+            </div>
+            {{ comment.body|linebreaks }}
+          </div>
+
+        {% if new_comment %}
+          <h2>Your comment has been added.</h2>
+          <!-- @todo: Ask @bpdesigns whether he designed those kinds of confirmations.
```

Comment:
@bpdesigns Remind me, do you have mockups for confirmations (like this)? If so, can you upload 'em to (or link them in) this issue? Thank you!